contracts/payloads/IGP80/PayloadIGP80.sol

```diff
@@ -0,0 +1,583 @@
+                TEAM_MULTISIG, 
+                true
+            );
+        }
+
+        {
+            address USDC_cbBTC_VAULT = getVaultAddress(102);
+
+            // [TYPE 1] USDC<>cbBTC | collateral & debt
+            Vault memory VAULT_USDC_cbBTC = Vault({
+                vault: USDC_cbBTC_VAULT,
+                vaultType: TYPE.TYPE_1,
+                supplyToken: USDC_ADDRESS,
+                borrowToken: cbBTC_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 8_000, // $8k
+                maxBorrowLimitInUSD: 10_000 // $10k
+            });
+
+            setVaultLimits(VAULT_USDC_cbBTC); // TYPE_1 => 102
+
+            VAULT_FACTORY.setVaultAuth(
+                USDC_cbBTC_VAULT, 
+                TEAM_MULTISIG, 
+                true
+            );
+        }
+    }
+
+    // @notice Action 2: Remove Multisig as auth from deUSD-USDC DEX
+    function action2() internal {
+
+        if (!PayloadIGP80(ADDRESS_THIS).skip_deusd_usdc_dex_auth_removal()) {
+            DEX_FACTORY.setDexAuth(
+                getDexAddress(deusd_usdc_dex_id),
+                TEAM_MULTISIG,
+                false
+            );
+        }
+    }
+
+    // @notice Action 3: Set dust allowance for USR-USDC DEX
+    function action3() internal {
+        address USR_USDC_DEX = getDexAddress(20);
+
+        {
+            // USR-USDC DEX
+            {
+                // USR-USDC Dex
+                Dex memory DEX_USR_USDC = Dex({
+                    dex: USR_USDC_DEX,
+                    tokenA: USR_ADDRESS,
+                    tokenB: USDC_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_USR_USDC); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(
+                    USR_USDC_DEX, 
+                    TEAM_MULTISIG, 
+                    true
+                );
+            }
+        }
+    }
+
+    // @notice Action 4:  Set dust allowance for sUSDe-USDT<>USDC-USDT T4 vault
+    function action4() internal {
+        address sUSDe_USDT_DEX_ADDRESS = getDexAddress(15);
+        address USDC_USDT_DEX_ADDRESS = getDexAddress(2);
+        address sUSDe_USDT__USDC_USDT_VAULT_ADDRESS = getVaultAddress(98);
+
+        {
+            // Update sUSDe-USDT<>USDC-USDT vault supply shares limit
+            IFluidAdminDex.UserSupplyConfig[]
+                memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+            config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: sUSDe_USDT__USDC_USDT_VAULT_ADDRESS,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimit: 10_000 * 1e18 // 10k shares
```

Comment:
if we want to set 10k USD then it would be 5k shares as 1 share = ~ 2 USD. same for other supply and borrow limits set in shares below